code/modules/recycling/sortingmachinery.dm

```diff
@@ -235,101 +255,173 @@
+
+	H.init(src)	// copy the contents of disposer to holder
+	air_contents = new() // The holder just took our gas; replace it
+	H.start(src) // start the holder processing movement
+	flushing = 0
+	// now reset disposal state
+	flush = 0
+	if(mode == 2)	// if was ready,
+		mode = 1	// switch to charging
+	update()
+	return
+
+/obj/machinery/disposal/deliveryChute/attackby(obj/item/I, mob/user, params)
+	if(!I || !user)
 		return
 
-	attackby(var/obj/item/I, var/mob/user, params)
-		if(!I || !user)
+	if(istype(I, /obj/item/weapon/screwdriver))
+		if(c_mode==0)
+			c_mode=1
+			playsound(src.loc, 'sound/items/Screwdriver.ogg', 50, 1)
+			to_chat(user, "You remove the screws around the power connection.")
+			return
+		else if(c_mode==1)
+			c_mode=0
+			playsound(src.loc, 'sound/items/Screwdriver.ogg', 50, 1)
+			to_chat(user, "You attach the screws around the power connection.")
+			return
+	else if(istype(I,/obj/item/weapon/weldingtool) && c_mode==1)
+		var/obj/item/weapon/weldingtool/W = I
+		if(W.remove_fuel(0,user))
+			playsound(src.loc, 'sound/items/Welder2.ogg', 100, 1)
+			to_chat(user, "You start slicing the floorweld off the delivery chute.")
+			if(do_after(user,20, target = src))
+				if(!src || !W.isOn()) return
+				to_chat(user, "You sliced the floorweld off the delivery chute.")
+				var/obj/structure/disposalconstruct/C = new (src.loc)
+				C.ptype = 8 // 8 =  Delivery chute
+				C.update()
+				C.anchored = 1
+				C.density = 1
+				qdel(src)
+			return
+		else
+			to_chat(user, "You need more welding fuel to complete this task.")
 			return
 
-		if(istype(I, /obj/item/weapon/screwdriver))
-			if(c_mode==0)
-				c_mode=1
-				playsound(src.loc, 'sound/items/Screwdriver.ogg', 50, 1)
-				to_chat(user, "You remove the screws around the power connection.")
-				return
-			else if(c_mode==1)
-				c_mode=0
-				playsound(src.loc, 'sound/items/Screwdriver.ogg', 50, 1)
-				to_chat(user, "You attach the screws around the power connection.")
-				return
-		else if(istype(I,/obj/item/weapon/weldingtool) && c_mode==1)
-			var/obj/item/weapon/weldingtool/W = I
-			if(W.remove_fuel(0,user))
-				playsound(src.loc, 'sound/items/Welder2.ogg', 100, 1)
-				to_chat(user, "You start slicing the floorweld off the delivery chute.")
-				if(do_after(user,20, target = src))
-					if(!src || !W.isOn()) return
-					to_chat(user, "You sliced the floorweld off the delivery chute.")
-					var/obj/structure/disposalconstruct/C = new (src.loc)
-					C.ptype = 8 // 8 =  Delivery chute
-					C.update()
-					C.anchored = 1
-					C.density = 1
-					qdel(src)
-				return
-			else
-				to_chat(user, "You need more welding fuel to complete this task.")
+
+/obj/item/shippingPackage
+	name = "Shipping package"
+	desc = "A pre-labeled package for shipping an item to coworkers."
+	icon = 'icons/obj/storage.dmi'
+	icon_state = "shippack"
+	var/obj/item/wrapped = null
+	var/sortTag = 0
+	var/sealed = 0
+
+/obj/item/shippingPackage/attackby(obj/O, mob/user, params)

```

Comment:
are there any objects you can put into one of these that can even be picked up...pretty sure it may be easier for you to just typecast this at `obj/item`.